src/main/scala/core/utils/Message.scala

```diff
@@ -0,0 +1,88 @@
+package core.utils
+
+import scala.collection.mutable.ArrayBuffer
+
+sealed trait Message extends Serializable
+
+object Message {
```

Comment:
As the Akka team recommends, each message should be declared in its' own receiver actor